services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -267,9 +250,32 @@ async def get_stack_status(self, node_uuid: NodeID) -> RunningDynamicServiceDeta
             service_message=container_message,
         )
 
-    async def _runner(self) -> None:
-        """This code runs under a lock and can safely change the SchedulerData of all entries"""
-        logger.debug("Observing dynamic-sidecars")
+    async def retrive(
```

Comment:
```suggestion
    async def retrieve(
```

Reply:
and actually would change the method name to something like retrieve_service_inputs.
retrieve on a scheduler does not mean much for me.
and again maybe question whether this call should really be a part of the scheduler. I mean the call does not need scheduling right?

Reply:
Will rename to `retrieve_service_inputs`, sounds better.
I'd keep this method attached to the scheduler. It keeps track of all the services. It has the necessary data required to call the dynamic-sidecar API. The same logic is applied to the `get_stack_status` call.